recipe/meta.yaml

```diff
@@ -16,25 +16,67 @@ source:
 
 build:
   skip: true  # [win]
-  number: 4
+  number: 5
   detect_binary_files_with_prefix: true
 
 test:
   files:
     - run_test.sh         # [osx]
     - run_test.py         # [osx]
   commands:
-    - test -f ${PREFIX}/lib/libncurses.a
-    - test -f ${PREFIX}/lib/libtinfo.a
-    - test -f ${PREFIX}/lib/libform.a
-    - test -f ${PREFIX}/lib/libmenu.a
-    - test -f ${PREFIX}/lib/libpanel.a
-    - test -f ${PREFIX}/lib/libncursesw.a
-    - test -f ${PREFIX}/lib/libtinfow.a
-    - test -f ${PREFIX}/lib/libformw.a
-    - test -f ${PREFIX}/lib/libmenuw.a
-    - test -f ${PREFIX}/lib/libpanelw.a
+    # Test libraries
+    {% set ncurses_libraries = [
+        "libncurses",
+        "libtinfo",
+        "libform",
+        "libmenu",
+        "libpanel",
+        "libncursesw",
+        "libtinfow",
+        "libformw",
+        "libmenuw",
+        "libpanelw"
+    ] %}
+    {% for each_ncurses_library in ncurses_libraries %}
+    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.a
+    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.dylib   # [osx]
+    - test -f ${PREFIX}/lib/{{ each_ncurses_library }}.so      # [linux]
+    {% endfor %}
 
+    # Test include directories
+    - test -d ${PREFIX}/include/ncurses
+    - test -d ${PREFIX}/include/ncursesw
+
+    # Test headers
+    {% set ncurses_headers = [
+        "curses.h",
+        "cursesapp.h",
+        "cursesf.h",
+        "cursesm.h",
+        "cursesp.h",
+        "cursesw.h",
+        "cursslk.h",
+        "eti.h",
+        "etip.h",
+        "form.h",
+        "menu.h",
+        "nc_tparm.h",
+        "ncurses.h",
+        "ncurses_dll.h",
+        "panel.h",
+        "term.h",
+        "term_entry.h",
+        "termcap.h",
+        "tic.h",
+        "unctrl.h"
+    ] %}
+    {% for each_ncurses_header in ncurses_headers %}
+    - test -L ${PREFIX}/include/ncurses/{{ each_ncurses_header }}
+    - test -L ${PREFIX}/include/ncursesw/{{ each_ncurses_header }}
+    - test -f ${PREFIX}/include/{{ each_ncurses_header }}
+    {% endfor %}
```

Comment:
Verify all the headers are present and that symlinks are used for the headers in the subdirectories.